.github/workflows/unitTest.yaml

```diff
@@ -0,0 +1,34 @@
+name: Run Unit Tests
+
+on:
+  push:
+    branches: [ develop ]
+    paths-ignore: [ '*.md' ]
+  pull_request:
+    branches: [ '**' ]
+
+jobs:
+  run-unit-tests:
```

Comment:
Maybe this should use the CI workflow from the template project? https://github.com/DataBiosphere/terra-java-project-template/blob/main/.github/workflows/build-and-test.yml

The template workflow runs build separately from test.

Maybe that can wait though; we'd also want to get sonar working with coverage.

Reply:
This was just a quick experiment to see if I could get the tests to run and pass at all. Agree it should use the standard CI workflow

Reply:
After looking some more I don't think it makes sense to use an exact copy paste of the template repo ci workflow here. It includes some additional logic which isn't relevant for stairway or other libraries such as running integration tests with test runner and building a docker image.

Running a build of the binary and the unit test suite in parallel does make sense to me though.

Reply:
Also agree we should have sonar here but I think that needs to be setup separately by infosec.